xmipptomo/protocols/protocol_deep_misalignment_detection.py

```diff
@@ -0,0 +1,423 @@
+# **************************************************************************
+# *
+# * Authors:     Federico P. de Isidro Gomez (fp.deisidro@cnb.csic.es) [1]
+# *
+# * [1] Centro Nacional de Biotecnologia, CSIC, Spain
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+import os
+
+from pwem.emlib import MetaData, MDL_MAX, MDL_MIN
+from pwem.protocols import EMProtocol
+from pyworkflow import BETA
+from pyworkflow.object import Set
+from pyworkflow.protocol import PointerParam, EnumParam, FloatParam, BooleanParam, LEVEL_ADVANCED
+from tomo.objects import SetOfCoordinates3D, SetOfTomograms, Coordinate3D, SubTomogram
+from tomo.protocols import ProtTomoBase
+from tomo import constants
+from xmipptomo import utils
+
+COORDINATES_FILE_NAME = 'subtomo_coords.xmd'
+BOX_SIZE = 32
+TARGET_SAMPLING_RATE = 6.25
+
+
+class XmippProtDeepDetectMisalignment(EMProtocol, ProtTomoBase):
+    """
+    Wrapper protocol to Xmipp image peak high contrast applied to any volume
```

Comment:
Update the help according to the protocol

Reply:
done